apps/pigment-css-next-app/src/app/material-ui/page.tsx

```diff
@@ -3,17 +3,16 @@ import * as React from 'react';
 import Link from 'next/link';
 import fs from 'fs/promises';
 import path from 'path';
-import { css } from '@pigment-css/react';
-import Box from '@pigment-css/react/Box';
+import { css } from '@mui/material-pigment-css';
 
 export default async function MaterialUIPage() {
   const rootPaths = await fs.readdir(path.join(process.cwd(), `src/app/material-ui`));
   return (
     <div>
       <h1>Material UI Components</h1>
       <nav>
-        <Box
-          as="ul"
+        <ul
+          // @ts-ignore
```

Comment:
Can we configure the module augmentation on the demo? In case people use this as a reference.